tests/functional/tests/rbd-mirror/test_rbd_mirror.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+
+
+class TestRbdMirrors(object):
+
+    @pytest.mark.no_docker
+    def test_rbd_mirror_is_installed(self, node, host):
+        assert host.package("rbd-mirror").is_installed
+
+    def test_rbd_mirror_service_is_running(self, node, host):
+        service_name = "ceph-rbd-mirror@rbd-mirror.{hostname}".format(
+            hostname=node["vars"]["inventory_hostname"]
+        )
+        assert host.service(service_name).is_running
+
+    def test_rbd_mirror_service_is_enabled(self, node, host):
+        service_name = "ceph-rbd-mirror@rbd-mirror.{hostname}".format(
+            hostname=node["vars"]["inventory_hostname"]
+        )
+        assert host.service(service_name).is_enabled
+
+    @pytest.mark.no_docker
+    def test_rbd_mirror_is_up(self, node, host):
+        cmd = "sudo ceph --name client.bootstrap-rbd --keyring /var/lib/ceph/bootstrap-rbd/{cluster}.keyring --cluster={cluster} --connect-timeout 5 -f json -s | python -c 'import sys, json; print \"\n\".join([i for i in json.load(sys.stdin)[\"servicemap\"][\"services\"][\"rbd-mirror\"][\"daemons\"] if i == \"{hostname}\"])'".format(
+            hostname=node["vars"]["inventory_hostname"],
+            cluster=node["cluster_name"]
+        )
+        output = host.check_output(cmd)
+        phrase = "{hostname}".format(hostname=node["vars"]["inventory_hostname"])
+        assert phrase in output
+
+    @pytest.mark.docker
+    def test_rbd_mirror_is_up(self, node, host):
```

Comment:
`test_rbd_mirror_is_up()` is redefined here, it means it will never be tested on non-containerized deployment.